manifest.md

```diff
@@ -139,7 +139,7 @@ Unlike the [Manifest List](#manifest-list), which contains information about a s
 
 - **`layers`** *array*
 
-    Each item in the array MUST be a [descriptor](descriptor.md).
+    This property MAY be empty, an image can have an empty layers array. Each item in the array MUST be a [descriptor](descriptor.md).
```

Comment:
Let's not mangle this line. The fact that these are descriptors is way more important than whether or not it can be empty.